src/components/CloseIcon.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const CloseIcon = () => (
+  <svg viewBox="0 0 32 32" aria-labelledby="close-icon">
```

Comment:
I love using SVGs 💛. It's only one suggestion **(no pressure to do it 😄).** In the last PR, we have created a folder `assets` where we put an SVG file (default image). I think we can put this SVG and other SVG files there, and import them.